templates/takeovers/_ubuntu-masters-virtual-202004.html

```diff
@@ -0,0 +1,15 @@
+{% with title="Join the virtual Ubuntu Masters event",
+subtitle="Join the demos and interactive sessions with leading technologists using Ubuntu on 29 April 2020",
```

Comment:
I think it's on the 30th of April

Reply:
they must have changed the date ... thanks for checking.